README.md

```diff
@@ -89,3 +92,12 @@ for additional information.
   $ mkdir /some/dir/nexus-data && chown -R 200 /some/dir/nexus-data
   $ docker run -d -p 8081:8081 --name nexus -v /some/dir/nexus-data:/nexus-data sonatype/docker-nexus3
   ```
+
+## Getting Help
+
+Looking to contribute to our Docker image but need some help? There's a few ways to get information or our attention:
+
+* File a public issue [here on GitHub](https://github.com/sonatype/docker-nexus3/issues)
```

Comment:
Is it worth differentiating between docker image issues and NX issues (which should be in JIRA)? Maybe solve that later if there's a deluge of wrong-place filings.